openfisca_us/variables/gov/ssa/ssi/eligibility/income/deemed/ssi_ineligible_parent_allocation.py

```diff
@@ -0,0 +1,28 @@
+from openfisca_us.model_api import *
+from openfisca_us.variables.gov.ssa.ssi.eligibility.income._apply_ssi_exclusions import (
+    _apply_ssi_exclusions,
+)
+
+
+class ssi_ineligible_parent_allocation(Variable):
+    value_type = float
+    entity = Person
+    label = "SSI ineligible parent allocation"
+    unit = USD
+    documentation = "The amount of income that SSI deems ought to be spent on this parent, and therefore is not deemed to SSI claimants."
```

Comment:
Removing speculation about intent.
```suggestion
    documentation = "The amount of income that SSI does not deem to SSI claimants."
```